azure-devops/azext_devops/dev/repos/pull_request.py

```diff
@@ -82,10 +78,6 @@ def list_pull_requests(repository=None, creator=None, include_links=False, revie
     :type skip: int
     :param top: Maximum number of pull requests to list.
     :type top: int
```

Comment:
Project help text instance here in list_pull_requests command. 

Probably a one off miss but difficult to find in the diff if any more are missed. Just double check once before merging.

Reply:
agree, lets clean/fix it as and when we find anything